releases/2.13.12.md

```diff
@@ -0,0 +1,66 @@
+* Warn about `@elidable` under `-Xsource:3` ([#10377](https://github.com/scala/scala/pull/10377) by [@som-snytt](https://github.com/som-snytt))
+
+### Collections
+
+* `IndexedSeq#head` now throws `NoSuchElementException` (not `IndexOutOfBoundsException`) ([#10392](https://github.com/scala/scala/pull/10392) by [@som-snytt](https://github.com/som-snytt))
+
+### Regression fixes
+
+* Deduplicate `JavaDeprecatedAttr` in `ClassfileParser` ([#10422](https://github.com/scala/scala/pull/10422) by [@som-snytt](https://github.com/som-snytt))
+* Avoid brittle flags encoding for Java enums ([#10424](https://github.com/scala/scala/pull/10424) by [@som-snytt](https://github.com/som-snytt))
+
+### Tooling
+
+The following change is mainly of concern to tooling maintainers.
+
+* In-source sbt compiler bridge ([#10472](https://github.com/scala/scala/pull/10472) by [@lrytz](https://github.com/lrytz))
+
+### Other notable changes
+
+For the complete 2.13.12 change lists, see [all merged PRs](https://github.com/scala/scala/pulls?q=is%3Amerged%20milestone%3A2.13.12) and [all closed bugs](https://github.com/scala/bug/issues?utf8=%E2%9C%93&q=is%3Aclosed+milestone%3A2.13.12).
+
+## Compatibility
+
+As usual for our minor releases, Scala 2.13.12 is [binary-compatible](https://docs.scala-lang.org/overviews/core/binary-compatibility-of-scala-releases.html) with the whole Scala 2.13 series.
+
+Upgrading from 2.12? Enable `-Xmigration` while upgrading to request migration advice from the compiler.
+
+## Contributors
+
+A big thank you to everyone who's helped improve Scala by reporting bugs, improving our documentation, spreading kindness in discussions around Scala, and submitting and reviewing pull requests! You are all magnificent.
+
+We especially acknowledge and thank A. P. Marki, also known as Som Snytt, who is responsible for a large share of the improvements in this release.
+
+This release was brought to you by 12 contributors, according to `git shortlog -sn --no-merges @ ^v2.13.11 ^2.12.x ^366ba2f`. Thank you A. P. Marki, Lukas Rytz, Seth Tisue, Scala Steward, Michel Davit, Nicolas Stucki, Alex Leigh, Eugene Yokota, Georgi Krastev, Jamie Willis, Julien Richard-Foy, NthPortal.
+
+Thanks to [Lightbend](https://www.lightbend.com/scala) for their continued sponsorship of the Scala 2 team’s efforts. Lightbend offers [commercial support](https://www.lightbend.com/lightbend-platform-subscription) for Scala.
+
+## Scala 2.13 notes
+
+The [release notes for Scala 2.13.0](https://github.com/scala/scala/releases/v2.13.0) have important information applicable to the whole 2.13 series.
+
+## Obtaining Scala
+
+Scala releases are available through a variety of channels, including (but not limited to):
+
+* Bump the `using scala` setting in your Scala CLI project
+* Bump the `scalaVersion` setting in your sbt project
```

Comment:
Let's also guide all those Mill users out there.

```suggestion
* Bump the `scalaVersion` setting in your sbt project
* Bump the `scalaVersion` setting/target in your Mill project
```

Reply:
"... in the build tool of your choice ..." or "... respectively, what they use at the office."

Reply:
I went with "sbt or Mill". We do have the "(but not limited to):" disclaimer.